### DISCUSSION TITLE
onRenderSuffix prop  equivalent for comboBox?

### DISCUSSION THREAD

**Discussion by chrichung** [microsoft/fluentui#22195] [General] [answered]
TextField has a prop for onRenderSuffix where I can render a Spinner component to indicate that a save call is being made. However, comboBox doesn't have something like that from looking at the [IComboBoxProps interface](https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox). Any idea how I can implement the following for comboBox?

`onRenderSuffix: () => {
            return (
                <>
                    {isSubmitting && (
                        <TooltipHost
                            id="goal-saving-id"
                            content={getLocalizedString(ControlStringConstants.savingButton)}
                        >
                            <Spinner aria-describedby="goal-saving-id" />
                        </TooltipHost>
                    )}
                </>
            );
        }`

**Comment by micahgodbolt** [comment, marked as answer]
moved to issue as a feature request.
